lib/jobs/isc-dhcp-lease-poller.js

```diff
@@ -84,14 +84,16 @@ function iscDhcpLeasePollerFactory(
 
         Promise.try(function () {
             var lease = self.parseLeaseData(data.toString());
-
-            self.queue.enqueue(
-                lookupService.setIpAddress.bind(
-                    lookupService,
-                    lease.ip,
-                    lease.mac
-                )
-            );
+            
+            if (!_.isUndefined(lease)) {
+                self.queue.enqueue(
+                    lookupService.setIpAddress.bind(
+                        lookupService,
+                        lease.ip,
+                        lease.mac
+                    )
+                );
+            }
```

Comment:
@heckj I think no, there are valid stanzas that we parse that don't have a mac address in them, and the way this code was written, it detects those by having an undefined lease. The bug here is more that we try to do things with an undefined lease instead of ignoring it. The unit tests should take care of all parsing cases we care about.